CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+# Contributing to the Mila Docs
+
+Thank you for your interest into making a better documentation for all at Mila. Here are some guidelines to help bring your contributions to life.
+
+## What could be included

```

Comment:
p-être simplement renommer "example contributions" et ajouter un petit peu de texte "encourageant" le partage de bonnes idées.

Reply:
Not sure where the example contributions goes but I can think of something more encouraging although I though there was encouragement below.